### PR TITLE
Laura edits pmr 2

### PR DESCRIPTION
**Pull request by laurapacilio**
This PR contains extensive edits to the private module registry section of the Terraform Cloud documentation. Specifically I:
- Updated all images for accuracy (the UI has changed) and to adhere to the learn style guide. 
- Copyedited and corrected the text for inaccuracies. Some content has been reordered to help with flow. For example, on two of the pages, information was pulled out if the introduction/workflow overview sections and placed in areas that align more with user tasks. The goal is to help users see this information when they need it. 
- Corrected some spacing issues and weird errors resulting from the GitHub UI in a previous PR.

I have:
- [x] Built this locally.
- [x] Assigned at least one reviewer.
- [x] Added at least one label.

### CODE REVIEW

File: content/source/docs/cloud/registry/design.html.md [hunk 1]
```diff
@@ -7,19 +7,13 @@ description: |-
...
-
-Once you finish, the designer returns the text of a `main.tf` configuration, which you must copy and paste to create a new VCS repo for workspaces. The designer does not automatically create any repos or workspaces; it's only a shortcut for writing Terraform code faster.
+The configuration designer lets you outline a configuration for a new workspace by choosing any number of private modules. It then lists those modules' variables as a fillable HTML form, with a helper interface for finding values that you can interpolate. When you are finished, the designer returns the text of a `main.tf` configuration. This is the same Terraform code you would have written in your text editor.
 
 ## Accessing the Configuration Designer
 
-Navigate to the modules list with the top-level "Modules" button, then click the "+ Design Configuration" button in the upper right.
+Click the "Registry" button in the main navigation bar and then click "<> Design Configuration".
 
 ![Terraform Cloud screenshot: the design configuration button](./images/design-start.png)
```
Comment (im2nguyen) [inline content/source/docs/cloud/registry/design.html.md:19]: should we wrap "Select Modules" in quotes?

> The Select Modules page appears.

File: content/source/docs/cloud/registry/design.html.md [hunk 2]
```diff
@@ -30,43 +24,52 @@ The Select Modules page appears.
 
 ## Adding Modules
 
-The left side of the Select Modules page has a filterable and searchable list of your organization's private modules. Choose any number of modules you want to add to your configuration.
+The left side of the Select Modules page has a filterable and searchable list of your organization's private modules that you can add to your configuration.
```
Comment (im2nguyen) [inline content/source/docs/cloud/registry/design.html.md:27]:
```suggestion
The left side of the "Select Modules" page has a filterable and searchable list of your organization's private modules that you can add to your configuration.
```

File: content/source/docs/cloud/registry/design.html.md [hunk 3]
```diff
@@ -30,43 +24,52 @@ The Select Modules page appears.
...
 
-Click the "Add Module" button for all of the modules you want to use in your configuration. These modules appear in the Selected Modules list on the right side of the page.
+Click "Add Module" for all of the modules you want to use in your configuration. These modules appear in the Selected Modules list on the right side of the page.
 
 ### Setting Versions
 
-By default, selecting a module adds its most recent version to the configuration. You can specify a different version by clicking the module's version number in the "Selected Modules" list on the right, which reveals a drop-down menu of available versions.
+Selecting a module adds its most recent version to the configuration. To specify a different version:
+
+1. Click the module's version number in the Selected Modules list on the right.
```
Comment (im2nguyen) [inline content/source/docs/cloud/registry/design.html.md:35]:
```suggestion
1. Click the module's version number from the "Selected Modules" list on the right.
```

File: content/source/docs/cloud/registry/design.html.md [hunk 4]
```diff
@@ -30,43 +24,52 @@ The Select Modules page appears.
...
+Sometimes, configuration users should be able to set certain variables according to their use cases.
+
+Select the "Deferred" checkbox to delegate a variable to configuration users. This ties the variable's value to a new top-level Terraform variable with no default value. All users that create a workspace from your configuration will have to provide a value for that variable.
 
 ## The Output Configuration
 
-When you've finished setting variables, click the "Next" button to view the completed output of the configuration designer.
+When all modules are configured, click "Next »".
+
+The Publish to VCS page appears. Use the "Preview configuration" dropdown to review the generated code.
```
Comment (im2nguyen) [inline content/source/docs/cloud/registry/design.html.md:69]:
```suggestion
The "Publish to VCS" page appears. Use the "Preview configuration" dropdown to review the generated code.
```

File: content/source/docs/cloud/registry/index.html.md [hunk 5]
```diff
@@ -7,15 +7,13 @@ description: |-
 
 # Private Module Registry
 
-> **Hands-on:** Try the [Add Public Modules to your Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry-add?in=terraform/modules&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
-
-> **Hands-on:** Try the [Share Modules in the Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry?in=terraform/modules&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
+> **Hands-on:** Try the [Add Public Modules to your Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry-add?in=terraform/modules&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial and [Share Modules in the Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry?in=terraform/modules&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorials on HashiCorp Learn.
```
Comment (im2nguyen) [inline content/source/docs/cloud/registry/index.html.md:10]: i think we cna remove these utm tags now

```suggestion
> **Hands-on:** Try the [Add Public Modules to your Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry-add?in=terraform/modules) tutorial and [Share Modules in the Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry?in=terraform/modules) tutorials on HashiCorp Learn.
```

File: content/source/docs/cloud/registry/publish.html.md [hunk 6]
```diff
@@ -11,109 +11,100 @@ description: |-
...
-     ![Terraform Cloud screenshot: the "registry" button and the "+Add Module" button](./images/publish-add-button.png)
+To publish a new module:
 
-    The Add Modules page appears with a list of available repositories.
+1. Click "Registry" in the main navigation bar. Then click "Publish" and select "Module" in the dropdown.
 
-    ![Terraform Cloud screenshot: the "add module" page, with a repository name entered](./images/publish-add-module.png)
+    ![Terraform Cloud screenshot: the "registry" button and the "+Add Module" button](./images/publish-add-button.png)
 
+      The Add Module page appears with a list of available repositories.
```
Comment (im2nguyen) [inline content/source/docs/cloud/registry/publish.html.md:58]:
```suggestion
      The "Add Module" page appears with a list of available repositories.
```

File: content/source/docs/cloud/registry/publish.html.md [hunk 7]
```diff
@@ -11,109 +11,100 @@ description: |-
...
 
+      ![Terraform Cloud screenshot: the "add module" page, with a repository name entered](./images/publish-add-module.png)
 
 2. Select the repository containing the module you want to publish.
 
     You can search the list by typing all part or all of a repository name into the filter field. Remember that VCS providers use `<NAMESPACE>/<REPO NAME>` strings to locate repositories. The namespace is an organization name for most providers, but Bitbucket Server (not Bitbucket Cloud) uses project keys, like `INFRA`.
```
Comment (im2nguyen) [inline content/source/docs/cloud/registry/publish.html.md:64]:
```suggestion
    You can search the list by typing part or all of a repository name into the filter field. Remember that VCS providers use `<NAMESPACE>/<REPO NAME>` strings to locate repositories. The namespace is an organization name for most providers, but Bitbucket Server (not Bitbucket Cloud) uses project keys, like `INFRA`.
```

File: content/source/docs/cloud/registry/publish.html.md [hunk 8]
```diff
@@ -11,109 +11,100 @@ description: |-
...
-To release a new version of a module, push a new version tag to its VCS repository. The registry will automatically import the new version.
+To release a new version of a module, push a new release tag to its VCS repository. The registry will automatically import the new version.
 
-Version tags must be a [semantic version](http://semver.org), which can optionally be prefixed with a `v`. For example, `v1.0.4` and `0.9.2`.
+Refer to [Preparing a Module Repository](#preparing-a-module-repository) for details about release tag requirements.
 
 ## Deleting Versions and Modules
 
-Each module's details page includes a "Delete Module 🗑" button, which can delete individual versions of a module or entire modules.
+You can delete individual versions of a module or entire modules. If a deletion would leave a module with no versions, Terraform Cloud will remove the entire module. To delete a module or version:
```
Comment (im2nguyen) [inline content/source/docs/cloud/registry/publish.html.md:80]:
```suggestion
You can delete individual versions of a module or the entire module. If deleting a module version would leave a module with no versions, Terraform Cloud will remove the entire module. To delete a module or version:
```

File: content/source/docs/cloud/registry/publish.html.md [hunk 9]
```diff
@@ -11,109 +11,100 @@ description: |-
...
-4. Select the desired action from the drop-down and confirm with the "Delete" button.
-    - "Delete only this module version" affects the version of the module you were viewing when you clicked delete.
-    - The other two options are only different if you have modules with the same name but different providers. (For example, if you have module repos named `terraform-aws-appserver` and `terraform-azure-appserver`, the registry treats them as alternate providers of the same `appserver` module.) If you use multi-provider modules like this, the "Delete all providers and versions for this module" option can delete multiple modules.
-5. Type the module's name to confirm the deletion.
+5. Type the module name and click "Delete".
+
+
+
+
+### Restoring a Deleted Module or Version
```
Comment (im2nguyen) [inline content/source/docs/cloud/registry/publish.html.md:97]:
```suggestion
### Restoring a Deleted Module or Version

```

**Review by im2nguyen — APPROVED** [review]
looks good aside from a few nits

File: content/source/docs/cloud/registry/design.html.md (hunk 1 above)
Comment (laurapacilio) [reply content/source/docs/cloud/registry/design.html.md:19]: Thank you for fixing everywhere :-) I see we do that on Learn.

**Comment by laurapacilio** [comment]
Ignoring broken links as it's just the SSL errors with learn tutorials that the web dev team says we can ignore.

**Pull request merged**